src/test_application/flight.cpp

```diff
@@ -121,15 +134,18 @@ void testapp::test_flight(std::unique_ptr<OSPMagnum>& pMagnumApp,
     osp::active::SysAreaAssociate::connect(rScene, rUni, areaSat);
 
     // Add default-constructed physics world to scene
-    rScene.get_registry().emplace<osp::active::ACompPhysicsWorld_t>(rScene.hier_get_root());
+    rScene.reg_emplace<osp::active::ACompPhysicsWorld_t>(rScene.hier_get_root());
 
     // Add a camera to the scene
 
     // Create the camera entity
     ActiveEnt camera = rScene.hier_create_child(rScene.hier_get_root(),
                                                        "Camera");
     auto &cameraTransform = rScene.reg_emplace<ACompTransform>(camera);
-    auto &cameraComp = rScene.get_registry().emplace<ACompCamera>(camera);
+    auto &cameraComp = rScene.reg_emplace<ACompCamera>(camera);
+    rScene.reg_emplace<ACompRenderingAgent>(camera, SysRender::get_default_rendertarget(rScene));
+    rScene.reg_emplace<ACompPerspective3DView>(camera, camera);
+    rScene.reg_emplace<ACompRenderer>(camera);
```

Comment:
It would really help to organize the setup in one readable section of code, such as:
```cpp
    // Create the camera entity
    ActiveEnt camera = rScene.hier_create_child(rScene.hier_get_root(),
                                                "Camera");
    auto &cameraTransform = rScene.reg_emplace<ACompTransform>(camera);
    auto &cameraComp = rScene.reg_emplace<ACompCamera>(camera);
    rScene.reg_emplace<ACompFloatingOrigin>(camera);
    cameraTransform.m_transform = Matrix4::translation(Vector3(0, 0, 25));
    cameraComp.m_far = 1u << 24;
    cameraComp.m_near = 1.0f;
    cameraComp.m_fov = 45.0_degf;
    cameraComp.calculate_projection();
    cameraComp.m_viewport
            = Vector2(Magnum::GL::defaultFramebuffer.viewport().size());

    // Add the debug camera controller to the scene. This adds controls
    auto camObj = std::make_unique<DebugCameraController>(rScene, camera);

    // Add a ACompDebugObject to camera to manage camObj's lifetime
    rScene.reg_emplace<ACompDebugObject>(camera, std::move(camObj));

    // Setup default render system
    osp::active::SysRender::setup(rScene);
    load_shaders(rScene);
    
    // Setup scene renderer, added to the camera component
    rScene.reg_emplace<ACompRenderingAgent>(camera, SysRender::get_default_rendertarget(rScene));
    rScene.reg_emplace<ACompPerspective3DView>(camera, camera);
    rScene.reg_emplace<ACompRenderer>(camera);
```

Reply:
Sounds good, I will make that change and then merge